# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1709828284 -3600
# Parent  9486e031bf6944876da2a30c9991ca9aae8e52a1
Feature: Linux systray: Patch app-indicator.c so it doesn't require root privileges.

diff --git a/third_party/appindicator/app-indicator.c b/third_party/appindicator/app-indicator.c
--- a/third_party/appindicator/app-indicator.c
+++ b/third_party/appindicator/app-indicator.c
@@ -1528,9 +1528,13 @@ check_connect (AppIndicator *self)
         g_free(name);
     }
 
     /* Do we have enough information? */
-    if (priv->menu == NULL) return;
+    /* See:
+     * https://github.com/Betterbird/thunderbird-patches/issues/20#issuecomment-1382846418
+     * https://bugs.kde.org/show_bug.cgi?id=464264#c9
+   	if (priv->menu == NULL) return;
+     */
     if (priv->icon_name == NULL) return;
     if (priv->id == NULL) return;
 
     if (priv->path == NULL) {
